tests/benchmark/utils.py

```diff
@@ -24,8 +24,8 @@
 def get_previous_dtype_result(dtype, sdtype, method, python_version=PYTHON_VERSION):
     """Return previous result for a given ``dtype`` and method."""
     data = get_previous_results()
-    df = data[python_version]
     try:
+        df = data[python_version]
```

Comment:
Why this change?

Reply:
If the previous result did not include data for a particular python version it would error out